test/test_rosbag/test/test_bag.py

```diff
@@ -188,6 +191,35 @@ def test_rosbag_filter(self):
 
         self.assertEquals(len(msgs), 5)
 
+    def test_trivial_rosbag_filter(self):
+        # regression test for issue #736
+        
+        try: 
+            tempdir = tempfile.mkdtemp()
```

Comment:
This line should be outside of the `try/except` block because if it raises there is nothing to cleanup yet.